tools/src/main/scala/scala/scalanative/build/NativeConfig.scala

```diff
@@ -136,7 +136,7 @@ sealed trait NativeConfig {
   def withOptimize(value: Boolean): NativeConfig
 
   /** Create a new config with given incrementalCompilation value */
-  def withIncrementalCompilation(value: Boolean): NativeConfig
+  def withUseIncrementalCompilation(value: Boolean): NativeConfig
```

Comment:
I guess we can remove `use` in the setter to make it more straightforward. The getter `useIncrementalCompilation` is fine, and should not be changed
```suggestion
  def withIncrementalCompilation(value: Boolean): NativeConfig
```